README.md

```diff
@@ -62,6 +57,48 @@ Objectives:
 - Extensible: Customizable without touching the core code.
 - Unified: Single codebase, deployable as an agent or collector with support for traces, metrics and logs.
 
+## Status
+
+The collector components and implementation are in different stages of stability, and usually split between
+functionality and configuration:
+
+| Component | Status |
+|---|---|
+| OTLP traces protocol | Stable |
+| OTLP metrics protocol | Stable |
+| OTLP logs protocol | Beta |
+| OTLP traces receiver functionality | Stable |
+| OTLP metrics receiver functionality | Stable |
+| OTLP logs receiver functionality | Beta |
+| OTLP traces receiver configuration | Beta |
+| OTLP metrics receiver configuration | Beta |
+| OTLP logs receiver configuration | Beta |
+| OTLP traces exporter functionality | Stable |
+| OTLP metrics exporter functionality | Stable |
+| OTLP logs exporter functionality | Beta |
+| OTLP traces exporter configuration | Beta |
+| OTLP metrics exporter configuration | Beta |
+| OTLP logs exporter configuration | Beta |
+| Logging exporter | Experimental |
+| Batch processor functionality | Beta |
+| Batch processor configuration | Beta |
+| MemoryLimiter processor functionality | Beta |
+| MemoryLimiter processor configuration | Beta |
+
+The collector public APIs are in different stages of stability (see more details about the [versioning](VERSIONING.md)):
+
+| Package | Status |
+|---|---|
+| go.opentelemetry.io/collector/model | RC |
+| go.opentelemetry.io/collector | Beta |
+
+See more details about the status:
+
+* **Experimental** components are under design, and have not been added to the specification.
+* **Beta** components are released and available for beta testing.
+* **RC** components are released and available for final testing before being marked as stable.
+* **Stable** components are backwards compatible and covered under long term support.
+
```

Comment:
should these map to https://github.com/open-telemetry/community/blob/47813530864b9fe5a5146f466a58bd2bb94edc72/maturity-matrix.yaml#L57 as per the proto repo? https://github.com/open-telemetry/opentelemetry-proto#maturity-level

Reply:
@bogdandrutu ⬆️

Reply:
Done.